Source/SwiftLintFramework/Rules/RuleConfigurations/FileHeaderConfiguration.swift

```diff
@@ -27,13 +27,17 @@ public struct FileHeaderConfiguration: RuleConfiguration, Equatable {
         }
     }
 
+    /// Used for unit tests of SWIFTLINT_CURRENT_FILENAME when File.path is nil
+    internal let filenameForTest: String = "Default.swift"
```

Comment:
what do you think about creating a fixture file for testing this instead? Check https://github.com/realm/SwiftLint/pull/2204 for an example.